CONTRIBUTION.md

```diff
@@ -0,0 +1,184 @@
+# Contributing to Antenna
+
+First off, thanks for contributing!
+
+This is a set of guidelines for contributing to Antenna, which is hosted by the [Eclipse Foundation](https://github.com/eclipse) on Github.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Eclipse Contributor Agreement](#eclipse-contributor-agreement)
+  * [Contact](#contact)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Java Styleguide](#java-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+## Code of Conduct
+
+This project is hosted by the Eclipse foundation and hence adheres to the 
```

Comment:
This is pretty much a repetition of the sentence before the TOC. Maybe only keep either or?